podman-compose.yml

```diff
@@ -1,11 +1,27 @@
 version: '3'
 services:
-  mongo:
-    image: "mongo"
-    container_name: "nodegmp-nosql"
+  app:
+    container_name: my_app
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        NODE_ENV: production
     restart: always
-    environment:
-      - MONGO_INITDB_ROOT_USERNAME=root
-      - MONGO_INITDB_ROOT_PASSWORD=nodegmp
     ports:
-      - '27017:27017'
\ No newline at end of file
+      - "8000:8000"
+    environment:
+      NODE_ENV: production
+      MONGO_URI: mongodb+srv://yuliabarbashova:fTiOkB5mVjOlvJLb@cluster0.0k6us4d.mongodb.net/test
+    depends_on:
+      - mongo
+    networks:
```

Comment:
Did you forget to describe network in your podman-compose file?
When I try running docker, I see the following error:
``` 
service "app" refers to undefined network my_network: invalid compose project
```